contrib/inventory/ec2.py

```diff
@@ -628,29 +632,25 @@ def tags_match_filters(self, tags):
         if not self.ec2_instance_filters:
             return True
         match = self.stack_filters
```

Comment:
This is never used except to return at the end. Would it make more sense to make this a constant, since it's never modified.

Reply:
Good catch :+1: